pkg/virtual/initializingworkspaces/builder/build.go

```diff
@@ -322,37 +405,14 @@ func (a *apiSetRetriever) GetAPIDefinitionSet(ctx context.Context, key dynamicco
 		return nil, false, err
 	}
 
-	apiResourceSchema := &apisv1alpha1.APIResourceSchema{
-		Spec: apisv1alpha1.APIResourceSchemaSpec{
-			Group: crd.Spec.Group,
-			Names: crd.Spec.Names,
-			Scope: crd.Spec.Scope,
-		},
+	apiResourceSchema, err := apisv1alpha1.CRDToAPIResourceSchema(crd)
```

Comment:
this is heavy, far too heavy for the use-case here. And it already was before as far as I see. But hiding it in this innocent method is bad.

Can we pre-compute the schemas? They are just system CRD schemas anyway.

Reply:
or at the very least put it into a cache versioning by RV, and only convert on demand.

Reply:
This will be a TODO